client/users/ratings/ratings.js

```diff
@@ -2,6 +2,8 @@ import { Template } from 'meteor/templating';
 import { Currencies } from '../../../lib/database/Currencies.js';
 import { Ratings } from '../../../lib/database/Ratings.js';
 
+import '/imports/ui/stylesheets/lux.min.css';
```

Comment:
This is affecting the whole site because ratings.js is loaded on every page as it's in the client directory. When this module is transitioned to an import it won't do it.